t/lib/t/MusicBrainz/Server/Controller/Work/Create.pm

```diff
@@ -4,89 +4,134 @@ use warnings;
 
 use Test::Routine;
 use Test::More;
-use HTTP::Request::Common;
 use MusicBrainz::Server::Test qw( capture_edits html_ok );
-use List::AllUtils qw( nsort_by );
```

Comment:
Just for confirmation: It was used for the only call of `nsort_by` on `capture_edits` results in the whole codebase. Other `capture_edits` results never caused any problem. So it is indeed fine to drop `nsort_by` here.